tutorials/cpi-sentiment-analysis/cpi-sentiment-analysis.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Set up SAP Hybris Marketing Cloud integration with Twitter  
-description: The SAP Cloud Platform Integration package implements an integration of social media data into SAP Hybris Marketing Cloud. It allows you to load and analyze social media data from Twitter into your SAP Hybris Marketing Cloud system.
+title: How to setup SAP Cloud Platform Integration for Twitter using the SAP Hybris Marketing Cloud System
```

Comment:
A warning about this title.  The tutorial catalog (and all tutorial lists) truncate the title if it is too long.  Starting a title with "How to setup" is unnecessary.  Use "Setup" instead, and consider shortening the official title of the software to make it all fit.